pkg/kubeclient/client_test.go

```diff
@@ -0,0 +1,13 @@
+package kubeclient
+
+import (
+	"github.com/stretchr/testify/assert"
+	"k8s.io/client-go/kubernetes/fake"
+	"testing"
+)
+
+func TestNewKubeClient(t *testing.T) {
+	k := KubeClient{}
+	k.CoreClient = fake.NewSimpleClientset()
+	assert.NotNil(t, k.CoreClient, "Kubecore client shouldn't be nil")
```

Comment:
Nothing, but this is only scaffolding. There is no real boundary, but I would expect to have the actual k8s code and tests in CRT-72 instead of the scaffolding code. The goal here would be to just add the correct libs to the code and do as much as to confirm they are there and can be used.